configs/11.0/build.mk

```diff
@@ -1 +0,0 @@
-../10.0/build.mk
```

Comment:
Seems we this file is symlinking to 10.0 on AT next too, we need to fix this by linking with AT 11.0.